db/migrate/20170421154535_create_default_price_groups_for_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDefaultPriceGroupsForUsers < ActiveRecord::Migration
+  def up
+    User.find_each do |user|
+      # does nothing if user_based_price_groups is not turned on
+      user.create_default_price_group!
+    end
+  end
+  def down
```

Comment:
Not sure why codeclimate isn't picking it up, but I would expect it to complain about adding a space between the methods. Also, the newlines at the top and bottom of the class.